blueprint/core/blueprint_GenericEditor.cpp

```diff
@@ -86,17 +78,14 @@ namespace blueprint
         // Dispatch parameter value updates to the javascript engine at 30Hz
         throttleMap.throttle(parameterIndex, 1000.0 / 30.0, [=]() mutable {
             juce::MessageManager::callAsync([=]() mutable {
-                if (safeAppRoot)
```

Comment:
Ahh interesting, don't need this anymore because the appRoot doesn't get swapped out itself anymore. Nice!